lib/gap_intelligence/models/header.rb

```diff
@@ -0,0 +1,13 @@
+module GapIntelligence
+  class Header < Record
+    attribute :name
+
```

Comment:
Not yet...but when we add spec headers we will. I was envisioning a SpecHeader class which extends from Header, then that class can have the extra attributes, like position and unit.